charts/azure-k8s-metrics-adapter/README.md

```diff
@@ -0,0 +1,66 @@
+# Azure Kubernetes Metrics Adapter
+
+An implementation of the Kubernetes [Custom Metrics API and External Metrics API](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-metrics-apis) for Azure Services. 
+
+This adapter enables you to scale your [application deployment pods](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) running on [AKS](https://docs.microsoft.com/en-us/azure/aks/) using the [Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) (HPA) with metrics from Azure Resources (such as [Service Bus Queues](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues)) and custom metrics stored in Application Insights. Learn more about [using an HPA to autoscale with with external and custom metrics](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-metrics-not-related-to-kubernetes-objects).
+
+## Installing the Chart
+
+```sh
+helm install --name my-release charts/azure-k8s-metrics-adapter
+```
+
+## Uninstalling the Chart
+
+```sh
+helm delete my-release
+```
+
+## Configuration
+
+### All Available Configuration Values
+
+| Parameter | Description | Default |
+| --------- | ----------- | ------- |
+| `image.repository` | Image repository | `jsturtevant/azure-k8-metrics-adapter-amd64` |
+| `image.tag` | Image tag | `latest` |
+| `image.pullPolicy` | Image pull policy | `Always` |
+| `logLevel` | Log level for V logs | `2` |
+| `replicaCount`  | Number of azure-k8s-metrics-adapter replicas  | `1` |
+| `adapterSecurePort` | Port on which the adapter is listening | `6443` |
+| `apiServiceInsecureSkipTLSVerify` | Disables TLS certificate verification when communicating with the apiService | `true` |
```

Comment:
Would recommend being secure by default, but allow to be insecure when there is a need to?

Reply:
I'd favor maybe following the pattern of the Prometheus adapter in this case, https://github.com/helm/charts/tree/master/stable/prometheus-adapter?

Reply:
agreed on using prom adapter pattern.  There needs to be more information added to explain how to create a cert.  The default behavior currently follows the default behavior in the [API Server](https://github.com/kubernetes-incubator/metrics-server/tree/master/deploy/1.8%2B) which has the tls skip set

Reply:
For this PR we leave it as it stands and then for a future issue we handle this pattern to limit the scope of this PR